_layouts/default.html

```diff
@@ -7,7 +7,7 @@
     <script
       defer
       data-domain="janmastodon.cz"
-      src="https://plausible.io/js/script.outbound-links.js"
+      src="https://plausible.nolog.cz/js/plausible.js"
```

Comment:
Nechceš zachovat to sledování odchozích odkazů? Přijde mně to jako zajímavá statistika.

Reply:
Ani jsem si nevšiml že to náš plausible umí, ale vypadá to že jo.
Přehodím!

Reply:
Jojo, umí, viz https://plausible.io/docs/outbound-link-click-tracking. Ještě je potřeba nastavit custom goal, viz tu dokumentaci.